src/k8sconfiguration/setup.py

```diff
@@ -33,7 +33,7 @@
 ]
 
 # TODO: Add any additional SDK dependencies here
-DEPENDENCIES = []
+DEPENDENCIES = ["pycryptodome~=3.9.8", "paramiko~=2.7"]
```

Comment:
`azure-cli` has pinned [`paramiko==2.6.0`](https://github.com/Azure/azure-cli/blob/b64a1e8e3b45cf3ffdae115871e5ae75e7174636/src/azure-cli/requirements.py3.Linux.txt#L111), adding `paramiko~=2.7` here will result in downloading 2.7.x version in extension folder but not loaded as the dependency in azure-cli has higher priority when loading.

If paramiko 2.6.0 also works for you, you can remove paramiko here. Otherwise you should bump the version in azure-cli and update the minVersion in the extension metadata.

Reply:
Thanks for pointing out; updated to use version 2.6.0